src/stories/util.js

```diff
@@ -0,0 +1,13 @@
+import { IntlProvider } from "react-intl";
+import React from "react";
+
+export const withReactIntl = (Component, messages = {}) => {
```

Comment:
You have built this, but where is it used?

Reply:
Removed this, it's not getting used anywhere